### ISSUE TITLE
[NBug] Unable to parse git command output.

### ISSUE THREAD

**Issue by dwainew** [gitextensions/gitextensions#6673]
## Error Details
```
System.Exception: Unable to parse git command output.
   at GitCommands.GitModule.<GetRemotesAsync>g__ParseRemotes|155_0(IEnumerable`1 lines)
   at GitCommands.GitModule.<GetRemotesAsync>d__155.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitUI.BranchTreePanel.RepoObjectsTree.RemoteBranchTree.<LoadNodesAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitUI.BranchTreePanel.RepoObjectsTree.Tree.<ReloadNodesAsync>d__27.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitUI.BranchTreePanel.RepoObjectsTree.Tree.<<UICommands_PostRepositoryChanged>b__6_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitUI.ThreadHelper.<>c__DisplayClass13_0.<<FileAndForget>b__0>d.MoveNext()
```

## Additional information
opening a private github repo locally

## Environment
- Git Extensions 3.1.0.5877
- Build bb1687f6bd0450d0c1a2bae10a37c65493fc1294
- Git 2.21.0.windows.1
- Microsoft Windows NT 10.0.17134.0
- .NET Framework 4.7.3394.0
- DPI 96dpi (no scaling)

**Comment by RussKie** [comment]
Please fill in all required information and ping us back to re-open.

**Comment by gerhardol** [comment]
Duplicate of #6562 Fixed in 6580, in 3.1.1

**Comment by gerhardol** [comment]
This is possibly fixed in 3.2. Removing need-review
